documentation/sections/changelog.md

```diff
@@ -1,6 +1,14 @@
 ## Change Log
 
 ```
+releaseHeader('2017-02-15', '1.12.4', '1.12.3')
+```
+
+*   The `cake` commands have been updated, with new `watch` options for most tasks. Clone the [CoffeeScript repo](https://github.com/jashkenas/coffeescript) and run `cake` at the root of the repo to see the options.
```

Comment:
As someone who has worked on the CoffeeScript source code, this is easily the most exciting part of the changelog. But do "regular users" even care? I'm not sure that we should mention the changes to CoffeeScript's internal Cakefile here – might just be confusing?